charts/minio-client-util/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: minio-client-util
 description: A Helm chart to deploy s3 utility to remove apitestrig reports from minio
 type: application
-version: 1.0.0-P1
+version: 1.0.0
```

Comment:
1.0.0-P2

Reply:
this is not released so pointing to last released chart version